response_operations_ui/templates/collection-exercise.html

```diff
@@ -12,6 +12,13 @@ <h1 class="jupiter" name="page-ce-title">{{survey.shortName}} {{ce.exerciseRef}}
         </div>
       </div>
     {% endif %}
+    {% if ci_unlinked %}
```

Comment:
This all works fine, It maybe would be nice to eventually generalise the UI for these information panels so we don't just keep adding if statements

Reply:
Yes definitely, i'll try put up another PR for this.

Reply:
https://github.com/ONSdigital/response-operations-ui/pull/136